src/QueryRenderer.ts

```diff
@@ -327,6 +319,33 @@ class QueryRenderChild extends MarkdownRenderChild {
 
         link.setText(linkText);
 
+        // Go to the line the task is defined at
+        const vault = this.app.vault;
+        link.addEventListener('click', async () => {
+            const result = await getTaskLineAndFile(task, vault);
+            if (result) {
+                const [line, file] = result;
+                const leaf = this.app.workspace.getLeaf();
+                await leaf.openFile(file, { eState: { line: line } });
+            }
+        });
+
+        link.addEventListener('mousedown', async (ev: MouseEvent) => {
+            // Open in a new tab on middle-click.
+            // This distinction is not available in the 'click' event, so we handle the 'mousedown' event
+            // solely for this.
+            // (for regular left-click we prefer the 'click' event, and not to just do everything here, because
+            // the 'click' event is more generic for touch devices etc.)
```

Comment:
As I understand it, there is already an Obsidian standard about which modified keys to use for what purposes.
The discussion starts [here in #plugin-dev Discord](https://discord.com/channels/686053708261228577/840286264964022302/1076577570964897902).

Speaking from a possible of relative ignorance about this level of detail, but if possible, it seems a good idea to call `Keymap.isModEvent`, as recommended there...

Reply:
That's very good to know. Fixed.